addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxCubeBridgeHandler.java

```diff
@@ -992,4 +1006,16 @@ private void updateCubeState() {
     public boolean hasExcessDutyCycle() {
         return dutyCycle >= MAX_DUTY_CYCLE;
     }
+
+    public void backup() {
+        if (this.backup != 2) {
```

Comment:
it was like not starting a new backup if one was already in the middle of storing a message.
e.g.  if (!this.backup.equals(backupState.INPROGRESS)) 

But now thinking of it... it actually does not matter that much, It can just start again if by accident it is triggered multiple times. (this collision was actually rather rare as once in progress from H to L it takes few ms)

Hence I removed the this if all together